tools/quickstart/quickstart/validator_account.py

```diff
@@ -71,32 +68,6 @@ def make_required_dirs(base_dir, chain_dir):
     os.makedirs(os.path.join(base_dir, KEY_DIR, chain_dir), exist_ok=True)
 
 
-def setup_author_address(setup_name, base_dir):
```

Comment:
Do not remove, but add a comment that this can not be used right now because of a bug in parity and a link to the bug report.